dapr/proto/scheduler/scheduler.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package dapr.proto.scheduler.v1;
+
+import "google/protobuf/empty.proto";
+import "dapr/proto/runtime/v1/dapr.proto";
+
+service Scheduler {
+	// ConnectHost is used by the daprd sidecar to connect to the scheduler service.
+	// The sidecar will send a stream of RegisterRequest, UnregisterRequest and blank 
+	// health-check messages.
+	// In return, scheduler service will send a stream of blank ConnectServerStream messages,
+	// which will also act as health check messages.
+	rpc ConnectHost(ConnectClientStream) returns (ConnectServerStream) {}
+	// ScheduleJob is used by the daprd sidecar to schedule a job.
+	rpc ScheduleJob(ScheduledJob) returns (google.protobuf.Empty) {}
```

Comment:
Can we have a response type even if there is not attribute now? It is helpful for extensibility.

Reply:
Done